lib/pg_ha_migrations/blocking_database_transactions_reporter.rb

```diff
@@ -1,4 +1,5 @@
 require 'stringio'
+require 'active_support/core_ext/string/strip'
```

Comment:
Instead of adding this line, could you change line 27/28 to use `<<~eos` instead of `<<-eos.strip_heredoc`? This is a language feature since Ruby 2.3, so I think we can simplify.